frontend/src/models/BoardModel.js

```diff
@@ -0,0 +1,34 @@
+import Field from "./FieldModel";
+
+export default class BoardModel {
+  constructor(input) {
+    let currentRow;
+    this.rows = [];
+
+    for (let idx = 0; idx < input.length; idx++) {
```

Comment:
`const idx`

Reply:
When I change 'let' to 'const' I get eslint  warning.

Line 8:45:  'idx' is constant  no-const-assign

Shall I remove this rule from linter?

Reply:
I double checked it and it's fine. It should stay as it is, my bad.